filewriter_v2/README.md

```diff
@@ -0,0 +1,399 @@
+
+<!-- SIMPLON V2 is currently not ready for filewriter
+
+#### SIMPLON V2 Configuration Parameters
+
+The configuration interface is accessible at `http://<ADDRESS_OF_DCU>/api/v2/daq/config`.
+
+| Parameter | JSON Type | Default | Description |
+|-----------|------|---------|-------------|
+| `enabled` | boolean | `false` | Enabled state of the FileWriter. |
+| `image_id_offset` | number | `1` | Unsigned integer defining the first image number on the axis `/entry/image_id`. |
+| `name_pattern` | string | `"series_$id"` | Base name of the files. `$id` will be replaced by the series number, i.e. using the default, the generated files will have the names:<br> `series_<series_number>_master.h5`<br> `series_<series_number>_data_<file_number>.h5`|
+| `images_per_file` | number | `1000` |  Unsigned integer defining the maximum number of images stored in each data file. If set to `0`, all images are stored directly in the master file and no data files are created. If set to a value greater than zero, the images are stored in multiple data files.|
+| `v2_preview_enabled` | bool | `false` | Enabled state of the FileWriter V2 preview. <br>**NOTE:** This key will be replaced when FileWriter V2 is released. There will likely be a `format` key instead, in analogy to Stream V2. |
+-->
+
+
+### Example
+
+FileWriter V2 may be configured with cURL.
+
+```sh
+curl \
+    -X PUT \
+    -H "Content-Type: application/json" \
+    -d '{"mode": {"value": "enabled"}, "v2_preview_enabled": {"value": true}}' \
+    $ADDRESS_OF_DCU/filewriter/api/1.8.0/config
+```
+
+## Data Access
+
+Files are stored locally (in memory) on the DCU and may be accessed over HTTP. Files are **not** stored persistently and all data will be lost on reboot.
+
+```sh
+# list available files
+curl $ADDRESS_OF_DCU/filewriter/api/1.8.0/status/files
+
+# file access
+curl -O $ADDRESS_OF_DCU/data/<name_pattern>_master.h5
+curl -O $ADDRESS_OF_DCU/data/<name_pattern>_data_<file_number>.h5
+```
+
+For further details on available commands and status parameters please refer to the [SIMPLON API documentation](https://media.dectris.com/210607-DECTRIS-SIMPLON-API-Manual_EIGER2-chip-based_detectros.pdf).
+
+## Multi-Channel Data
+
+One of the main new features of FileWriter V2 is the support of multi-channel data, i.e. each image consists of multiple channels, one per enabled threshold/difference mode. The dataset `/entry/data/data` is four-dimensional with the shape `[nP, nC, i, j]`, the notation used is defined [here](#notation). The channels are identified via strings given in the `channel` [axis](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-field) of the _NXdata_ class[^2]. For every channel name listed in this field, there exists a subgroup of the [_NXdetector_]() group of type _NXdetector_channel_[^1] with the same name followed by the suffix `_channel`. This group contains channel-specific metadata, for example `threshold_energy`, `flatfield` or `pixel_mask`.
```

Comment:
You can drop the "the" before "shape".

Perhaps instead of "the notation used is defined here", the "Notation" subsection could be moved up and this remark could be avoided altogether? You probably considered this and couldn't find a good place for it? I don't have a good answer.

Reply:
Yes, it is as you suspected, I couldn't find a really good place for it.

Reply:
"the" removed in [92ac462](https://github.com/soph-dec/documentation/pull/1/commits/92ac462f59e9a78828e01acde45d262271abf443)